### PR TITLE
[ci skip] [skip ci] ***NO_CI*** Remove duplicate key

### PR DESCRIPTION
**Pull request by BastianZim**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Remove duplicate GitHub key which is currently holding up the python 3.11 migration: https://conda-forge.org/status/

### CODE REVIEW

**Comment by BastianZim** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ build:
   script:
     - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
     - {{ PYTHON }} -m pip install . -vv
-  number: 0
+  number: 1
```
Comment (weiji14) [inline recipe/meta.yaml:16]: Thanks @BastianZim for picking up the extra lines. Is a new build necessary though, or can we add `[skip ci]` to the title of this Pull Request to skip a new package build? If so, then the build number can be kept at 0.

Comment (BastianZim) [reply recipe/meta.yaml:16]: Both works. I just didn‘t do it because most are unfamiliar with that which makes reviews longer. I’ll change it. :)

**Review by weiji14 — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (weiji14) [reply recipe/meta.yaml]: Cool, I'll merge this once the tests pass (and make sure the `[ci skip]` is intact)!

**Pull request merged**